Packs/ZeroFox/Integrations/ZeroFox/ZeroFox.py

```diff
@@ -586,6 +631,24 @@ def get_cti_exploits(self, since: str) -> dict[str, Any]:
 """ HELPERS """
 
 
+class AlertToIncident:
+    def __init__(self):
+        self.integration_instance = demisto.integrationInstance()
+
+    def map(self, alert: dict[str, Any]) -> dict[str, Any]:
+        alert_id = str(alert.get("id", ""))
+        alert["mirror_direction"] = "In"
+        alert["mirror_instance"] = self.integration_instance
+        incident = {
+            "rawJSON": json.dumps(alert),
+            "name": f"ZeroFox Alert {alert_id}",
+            "dbotMirrorId": alert_id,
+            "occurred": alert.get("timestamp", ""),
+            "details": ""  # TODO
```

Comment:
Please remove if you don't use it.

Reply:
didn't get added to latest commit for some reason, added now